tjreports/site/models/reports.php

```diff
@@ -380,10 +626,36 @@ protected function getListQuery()
 
 		if (!empty($sortKey) && !in_array($sortKey, $this->sortableWoQuery))
 		{
-			$query->order($sortKey . ' ' . $orderDir);
+			$query->order($db->quoteName($sortKey) . ' ' . $orderDir);
 			$this->canLimitQuery = true;
 		}
 
+		// Joomla fields integration - Get custom fields data
+		// Proceed if table eixists, and at least one custom field is seleced for showing
+		if ($this->customFieldsTableExists && !empty($this->customFieldsTableColumnsForQuery) && !empty($this->customFieldsQueryJoinOn))
```

Comment:
`customFieldsQueryJoinOn` should also be declared as a class variable ?

Reply:
added